### PR TITLE
set JsonConfig configuration for gpcommonswiki

### PR DESCRIPTION
**Pull request by ugochimobi**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -1949,16 +1949,39 @@
...
 	],
 	'wgJsonConfigModels' => [
 		'default' => [
 			'Map.JsonConfig' => JsonConfig\JCMapDataContent::class,
 			'Tabular.JsonConfig' => JsonConfig\JCTabularContent::class,
 		],
+		'gpcommonswiki' => [
+			'Tabular.JsonConfig' => 'JsonConfig\JCTabularContent',
+			'Map.JsonConfig' => 'JsonConfig\JCMapDataContent',
+		],
```
Comment (Universal-Omega) [inline LocalSettings.php:1984]: This is the exact same as the above default.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (ugochimobi) [reply LocalSettings.php:1984]: > This is the exact same as the above default.

Oh, I'd remove that then, I thought there was a difference between both, like if they function differently.

Comment (Universal-Omega) [reply LocalSettings.php:1984]: Nope, its the exact same as the default.

Comment (ugochimobi) [reply LocalSettings.php:1984]: > Nope, its the exact same as the default.

Okay, I'd just remove it then.

Comment (Universal-Omega) [reply LocalSettings.php:1984]: Please do then.

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**